packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-block-convert-button.js

```diff
@@ -169,8 +169,7 @@ export default function ReusableBlockConvertButton( {
 					>
 						<VStack spacing="5">
 							<TextControl
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
Doesn't appear in the app anymore, but should be safe (same as https://github.com/WordPress/gutenberg/pull/64213#discussion_r1704383798).